src/Tools/GenerateGlobalAnalyzerConfigs/Program.cs

```diff
@@ -521,5 +554,13 @@ public Assembly LoadFromPath(string fullPath)
                 return Assembly.LoadFrom(fullPath);
             }
         }
+
+        // NOTE: **Do not** change the names of the fields for this enum - that would be a breaking change for user visible property setting for `AnalysisMode` property in MSBuild project file.
+        private enum AnalysisMode
+        {
+            Default,
+            AllDisabled,
+            AllEnabled,
```

Comment:
Changing the value to more correctly reflect the mode:
```suggestion
            AllDisabledByDefault,
            AllEnabledByDefault,
```